source/hello.js

```diff
@@ -10,10 +14,14 @@ export default React => stampit(React, {
   propTypes: {
     actions: PropTypes.shape({
       setWord: func.isRequired,
-      setMode: func.isRequired,
```

Comment:
The new .eslint file that @ericelliott added to the repo yesterday rejects commits if they have trailing commas.